.github/workflows/generate_readme_on_push.yml

```diff
@@ -0,0 +1,37 @@
+name: Generate README on Push
+
+on:
+  push:
+    branches: [ "fdsimms/chore-github-action" ]
```

Comment:
Make sure this branch changes to `master` (or `main` when #143 is done) before merging